pkg/net/port.go

```diff
@@ -42,3 +81,17 @@ func ParsePort(s string) (p Port, err error) {
 
 	return Port(i), nil
 }
+
+// ParsePort - parses uint16 into Port
+func ParsePortNumber(n uint16) (p Port, err error) {
```

Comment:
`Port` in go internals is string.  This function is not needed.  Also please make necessary changes to the fix to have fewer diffs and avoid unnecessary changes.